docs/tutorial/updates.md

```diff
@@ -261,7 +261,49 @@ server-communication aspect of the process by loading your update from a local d
 
 ## Update server specification
 
-A Squirrel-compatible update server has different
+For advanced deployment needs, you can also roll out your own Squirrel-compatible update server.
+Squirrel.Windows and Squirrel.Mac clients require different response formats,
+but you can use a single server for both platforms by sending requests to
+different endpoints depending on the value of `process.platform`.
+
+```js title='main.js'
+const { app, autoUpdater } = require('electron')
+
+const server = 'https://your-deployment-url.com'
+// e.g. for Windows and app version 1.2.3
+// https://your-deployment-url.com/update/win32/1.2.3
+const url = `${server}/update/${process.platform}/${app.getVersion()}`
+
+autoUpdater.setFeedURL({ url })
+```
+
+### Windows
+
+A Squirrel.Windows client expects the update server to return the `RELEASES` artifact
+of your build at the `/RELEASES` subpath of your endpoint.
+
+For example, if your feed URL is `https://your-deployment-url.com/update/win32/1.2.3`,
+then the `https://your-deployment-url.com/update/win32/1.2.3/RELEASES` endpoint
+should return the contents of the `RELEASES` artifact of the version you want to serve.
+
+```plaintext title='https://your-deployment-url.com/update/win32/1.2.3/RELEASES'
+B0892F3C7AC91D72A6271FF36905FEF8FE993520 https://your-static.storage/your-app-1.2.3-full.nupkg 103298365
+```
+
+### macOS
+
+A Squirrel.Mac client expects a JSON response at the feed URL's endpoint.
```

Comment:
This is the "server" method instead of the static JSON method, if we want to document this technique we should also clearly document that the server should respond with status `204 No Content` when no update is available

Reply:
[90b0b55](https://github.com/electron/electron/pull/43234/commits/90b0b55f82660b3295c70e286ab0768c8cdd0ed9)